src/renderer/pages/Widget/Header.tsx

```diff
@@ -149,8 +149,6 @@ export default function Header() {
       <div
         role="button"
         tabIndex={0}
-        onClick={window.electron.openSyncFolder}
-        onKeyPress={window.electron.openSyncFolder}
```

Comment:
I would say is better to remove the onClick from the child since this html item is the one with role="button"